packages/pn-commons/src/components/CustomDatePicker.tsx

```diff
@@ -1,12 +1,16 @@
+/* eslint-disable sonarjs/no-identical-functions */
```

Comment:
Use https://mui.com/x/react-date-pickers/date-picker/ that automatically switch between the two views

Reply:
done